### DISCUSSION TITLE
Ban Item Module

### DISCUSSION THREAD

**Discussion by tthetrault** [Zrips/CMI#5899] [Feature Requests]
Hi, i thought that maybe a new module or simple feature that can ban items easily with some flags (break,place,craft,etc) and show to players a proper list of them with a command could be really nice. No banitems plugins seem to have quite all the needed feature, especially for modded servers..

**Comment by ghost** [comment]
You can use WorldGuard's blacklisting features for that kind of item blacklisting.
I mention that since most servers will have WorldGuard installed.
